app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cash = {}
+
+    def wrapper(*args, **kwargs) -> int:
```

Comment:
You could use a decorator from functools to preserve the wrapped function name and docstring.